app.js

```diff
@@ -1,22 +1,61 @@
 const fs = require('fs');
+const path = require('path');
 const acorn = require('acorn');
 const escodegen = require('escodegen');
 const argv = require('boring')();
+const cp = require('child_process');
+const { stripIndent } = require('common-tags');
 
 // This is a random identifier not used anywhere else
 const blankLineMarker = "// X0k7FEu5a6!bC6mV";
 
-const moduleNames = argv._;
+try {
+  cp.execSync('git status', { encoding: 'utf8' });
+} catch (e) {
+  fail(stripIndent`
+    This project does not appear to be using git. For your protection and to simplify
```

Comment:
Because without `git mv` it is very confusing to look at what it did. And because using this without git is just a bad idea.